content/en/network_performance_monitoring/installation.md

```diff
@@ -235,8 +235,8 @@ If you already have the [Agent running with a manifest][3]:
                         readOnly: true
                       - name: debugfs
                         mountPath: /sys/kernel/debug
-                      - name: s6-run
-                        mountPath: /var/run/s6
+                      - name: sysprobe-socket-dir
```

Comment:
Since this `volumeMount` is renamed, is there a `volume` that also needs to be renamed?

Reply:
Good point. Updated.

Reply:
Is the `s6-run` volume still needed?

Reply:
I am not sure to be frank. I have not removed it to be consistent with https://docs.datadoghq.com/resources/yaml/datadog-agent-npm.yaml

Reply:
Looks like the linked YAML doesn't use it either. It can be removed in both places.

Reply:
Updated.

Reply:
@brycekahle, please have a look when you get a chance.